src/celengine/refcounted.cpp

```diff
@@ -0,0 +1,4 @@
+
+#include "refcounted.h"
+
+RefCounted::~RefCounted() {}
```

Comment:
and this is something absolutely weird, you mark `RefCounted::~RefCounted()` as 0 but here add an implementation

Reply:
I thought exactly the same, but without it there is a linker error. I found this solution on the Internet...